src/test/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/TransferIntoPrisonServiceTest.kt

```diff
@@ -160,6 +162,7 @@ internal class TransferIntoPrisonServiceTest {
           birthDate = LocalDate.now().minusYears(30)
           gender = Gender("M", "MALE")
         }
+        bookingBeginDate = LocalDateTime.now().minusDays(1)
```

Comment:
We can't have null booking begin date anymore, which aligns with NOMIS where that column is never null.